### PR TITLE
Ofir/znk category summary

### PR DESCRIPTION
**Pull request by OfirZinkerz**

### CODE REVIEW

File: .jshintrc [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "curly": true,
     "eqeqeq": true,
     "esnext": true,
-    "latedef": true,
+    "latedef": false,
```
Comment (assafshp) [inline .jshintrc]: late definition should be false, unless we get another

File: src/components/znkCategoryStats/module.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        'znk.infra.znkProgressBar',
+        'znk.infra.stats',
+        'znk.infra.contentGetters',
+        'znk.infra.general',
+        'znk.infra.svgIcon'
+    ])
+    .config([
+        'SvgIconSrvProvider',
+        function (SvgIconSrvProvider) {
+            var svgMap = {
```
Comment (assafshp) [inline src/components/znkCategoryStats/module.js]: ngInject

File: src/components/znkCategoryStats/znkCategoryStats.component.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+            controllerAs: 'vm',
+            controller: function (StatsSrv, CategoryService, $q) {
+                'ngInject';
+                var vm = this;
+                var PERCENTAGE = 100;
+                var MILLISECOND = 1000;
+
+                buildUiCategory(vm.categoryId);
+
+                function buildUiCategory(categoryId) {
```
Comment (assafshp) [inline src/components/znkCategoryStats/znkCategoryStats.component.js]: +categoryId ?

File: src/locale/locale-en.json [hunk 4]
```diff
@@ -100,5 +100,11 @@
         "CLOSE": "CLOSE",
         "PlEASE_COMPLETE_ASSIGNMENT": "Please complete this assignment before your next Zinkerz lesson",
         "YOUR_ASSIGNMENT_IS_LATE": "Your assignment is late"
+    },
+    "ZNK_CATEGORY_SUMMARY":{
+        "MASTERY": "mastery",
```
Comment (assafshp) [inline src/locale/locale-en.json]: mastery or acurcy?

File: .jshintrc (hunk 1 above)
Comment (OfirZinkerz) [inline .jshintrc]: Ok

File: src/components/znkCategoryStats/module.js (hunk 2 above)
Comment (OfirZinkerz) [inline src/components/znkCategoryStats/module.js]: fixed

File: src/components/znkCategoryStats/znkCategoryStats.component.js (hunk 3 above)
Comment (OfirZinkerz) [inline src/components/znkCategoryStats/znkCategoryStats.component.js]: no need

File: src/locale/locale-en.json (hunk 4 above)
Comment (OfirZinkerz) [inline src/locale/locale-en.json]: acurcy

**Pull request merged**